ui/webui/src/components/Error.jsx

```diff
@@ -97,14 +98,16 @@ export const BZReportModal = ({
               <Stack hasGutter>
                   <FormHelperText isHidden={false}>
                       <HelperText>
-                          <HelperTextItem>{_("Reporting an issue will send information over the network. Plese review and edit the attached log to remove any sensitive information.")}</HelperTextItem>
+                          {isConnected
+                              ? <HelperTextItem> {_("Reporting an issue will send information over the network. Please review and edit the attached log to remove any sensitive information.")} </HelperTextItem>
+                              : <HelperTextItem icon={<DisconnectedIcon />}> {_("Network not available. Configure the network in the top bar menu to report the issue.")} </HelperTextItem>}
```

Comment:
I think this is good for now in the Fedora Workstation environment, but we should take a note to handle this somehow better for other environments. Eq. boot.iso currently does not have a top bar & other Live spins could have different mechanisms for this.